src/components.jsx/Article.jsx

```diff
@@ -0,0 +1,87 @@
+import {
+  Avatar,
+  Box,
+  Button,
+  Card,
+  CardFooter,
+  Flex,
+  Heading,
+  Image,
+  Text,
+} from "@chakra-ui/react";
+import React, { useState } from "react";
+import getUser from "../utils/getUser";
+
+const Article = ({ article }) => {
+  const [articleAuthor, setArticleAuthor] = useState("");
+
+  getUser(article.author).then(({ user }) => {
+    setArticleAuthor(user);
+  });

```

Comment:
I would not do an API request invocation in a component. Every time this renders you will make a request which sets state, which causes a rerender which makes an api call which updates state.... you get the idea. Check your network tab, you're in an infinite loop.